### DISCUSSION TITLE
FBW A320 v0.6.0 or Dev Displays/alignment issue

### DISCUSSION THREAD

**Discussion by MTPiperPilot21** [flybywiresim/aircraft#4493] [Help]
Hello all,

Since switching over to the standalone FBW A320, I’ve been having an issue with the PFD & MFD showing the IRS/ADIRS is not being aligned when starting at the runway. Also get the “NAV TCAS FAULT” message. The TCAS panel is turned on correctly, based on switch position. But, other systems/switches are not configured correctly (lights, spoilers, etc).

It generally seems to follow the same pattern. First flight in the plane is fine, but if I got back to the main menu and do another flight, then I get the issue.

[https://forums.flightsimulator.com/uploads/default/original/3X/9/c/9c87f48d173c174f22c5c1ba72952ed4ed62e4e4.jpeg](url)

I cannot re-align or reconfigure anything when this happens. Switches throw, but nothing happens.

Nothing has changed in my Community folder except the following:

-Run FBW livery converter - seems to have worked correctly.
-Updated Mugz Copilot mod to latest version

**For troubleshooting, I’ve tried the following:** 

-Download Stable v0.6.0 files direct from FBW Github, also tried Dev. version Result: **No go.** Still causes avionics bug after first flight.

-Alternatively, install from the FBW installer Result: **No go.** Still causes avionics bug after first flight.

-Tried renaming FBW mod to place above liveries, and below in the Community folder Result: **No go.** Still causes avionics bug after first flight.

-Tried without any liveries in Community folder Result: **No go.** Still causes avionics bug after first flight.

-Tried without anything in Community folder, except FBW mod folder. Result: **No go.** Still causes avionics bug after first flight.

-Remove AI model folders, AI entries in aircraft.cfg’s and made new layout.json files for all my installed liveries. Using with a fresh FBW Stable v0.6.0 install. Launch MSFS and launch same flight multiple times.
**Result: No go.** Still causes avionics bug after first flight.

-Just to make sure, tried removing all liveries with a fresh FBW Stable v0.6.0 install. Launch MSFS and launch same flight multiple times.
**Result: No go.** Avionics bug appeared right from the get-go. So I think last test I was just lucky it didnt.

-And also, just to make sure, I again tried the Dev version I downloaded on 4/2/21 before the split to standalone. Launched MSFS and launched same flight multiple times.
**Result: No go.** Also had the bug appear this time. Again, previous testing was just lucky. _But I do know that this bug did not appear prior to the 1.15.7.0 (or 1.15.8.0) update. *I normally do short flights (30 min in the air or so), so I would sometimes do several back to back, and never had this issue._

-Then, I tried the Stable v0.6.0 version without the (updated) Copilot mod by Mugz.
**Result: No go.** Bug appeared after first successful flight launch.

-Lastly, I tried the same with the completely default, unmodded A320Neo. Issue did not appear. I tried 10 flight launches. **All good.**

So…to recap.

It would seem that, on my system at least, this bug is random (may or may not appear after one or two flights), isolated to the FBW A320 mods (pre-split version and new standalone Stable and Dev) and is not related to (properly converted) liveries, AI model folders inside said liveries, or to the Mugz Copilot mod. It does appear to be a new bug in the new 1.15.8.0 version of MSFS. The only workaround I have found is restarting MSFS between flights. 

Discussion that started on MSFS forums: [here](https://forums.flightsimulator.com/t/fbw-a320-v0-6-0-or-dev-displays-issue/393179 )

Hope this helps!

**Comment by BozoBBozo** [comment]
I have had this issue sporadically (maybe 1 in 8 times) in the past week, with the same version/configuration, and can't seem to identify the condition that causes this. I'm lazy and always spawn on the runway.
